frontend-nannyway/src/components/accordion/ChildInCareAccordion.js

```diff
@@ -57,6 +76,16 @@ export default function ChildInCareAccordion({kid}) {
         setExpanded(isExpanded ? panel : false);
     };
 
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
     return (
         <div className={classes.root}>
             <Accordion expanded={expanded === 'panel1'} onChange={handleChange('panel1')}
```

Comment:
why you dont use just true and false? do you plan to have more than one Accordion parts?

Reply:
what? sry, i don´t get your point. please write loud and slowly 😆

Reply:
Done